src/switch_core_media.c

```diff
@@ -9504,6 +9507,11 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_activate_rtp(switch_core_sessi
 			if (v_engine->tmmbr) {
 				flags[SWITCH_RTP_FLAG_TMMBR]++;
 			}
+
+			if (switch_channel_var_true(session->channel, "fire_rtcp_events") && 
```

Comment:
why are you checking the chan var two times ? should be only once.

Reply:
@dragos-oancea given the complexity of the method it's in, I thought it safest to put the check closely above the switch_rtp_new() call.  the way I did it, it's clear what applies to audio vs video vs text (no flag is set for text).  I'll combine them if that's better -- I think anywhere below switch_assert(session) will work, but let me know.